doc/devices/qpu.rst

```diff
@@ -64,33 +62,18 @@ These operations include:
     pennylane_forest.ops.ISWAP
     pennylane_forest.ops.PSWAP
 
-Device options
+QVM and quilc server configuration
 ~~~~~~~~~~~~~~
 
-On initialization, the PennyLane-Forest devices accept additional keyword 
-arguments beyond the PennyLane default device arguments.
-
-``forest_url`` (*str*)
-    the Forest URL server. Can also be set by
-    the environment variable ``FOREST_SERVER_URL``, or in the ``~/.qcs_config``
-    configuration file. Default value is ``"https://forest-server.qcs.rigetti.com"``.
-
-``qvm_url`` (*str*)
-    the QVM server URL. Can also be set by the environment
-    variable ``QVM_URL``, or in the ``~/.forest_config`` configuration file.
-    Default value is ``"http://127.0.0.1:5000"``.
-
-``compiler_url`` (*str*)
-    the compiler server URL. Can also be set by the environment
-    variable ``COMPILER_URL``, or in the ``~/.forest_config`` configuration file.
-    Default value is ``"http://127.0.0.1:6000"``.
```

Comment:
These can still be set by environment variables, so might be worth mentioning that.

Reply:
Ah, I did add a note for that but admittedly put it in a weird place. I placed it under the note about launching `quilc` and `qvm`. I reference them to this pyQuil documentation: https://pyquil-docs.rigetti.com/en/stable/advanced_usage.html#pyquil-configuration